ui/public/config.js

```diff
@@ -1,4 +1,5 @@
 window.config = {
   API_URL: 'https://api.hub.tekton.dev',
-  GH_CLIENT_ID: ''
+  GH_CLIENT_ID: '',
+  API_VERSION: 'v1'
```

Comment:
I think we shouldn't add this

Reply:
we should add this to keep `v1` as default ,I guess otherwise catalog wouldn't able to access